### ISSUE TITLE
Node.js Foundation Build WorkGroup Meeting 2018-07-10

### ISSUE THREAD

**Issue by mhdawson** [nodejs/build#1387]
## Time

**UTC Tue 03-Jul-2018 22:00 (10:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 10-Jul-2018 15:00 (03:00 PM) |
| US / Mountain | Tue 10-Jul-2018 16:00 (04:00 PM) |
| US / Central  | Tue 10-Jul-2018 17:00 (05:00 PM) |
| US / Eastern  | Tue 10-Jul-2018 18:00 (06:00 PM) |
| London        | Tue 10-Jul-2018 23:00 (11:00 PM) |
| Amsterdam     | Wed 11-Jul-2018 00:00 (12:00 AM) |
| Moscow        | Wed 11-Jul-2018 01:00 (01:00 AM) |
| Chennai       | Wed 11-Jul-2018 03:30 (03:30 AM) |
| Hangzhou      | Wed 11-Jul-2018 06:00 (06:00 AM) |
| Tokyo         | Wed 11-Jul-2018 07:00 (07:00 AM) |
| Sydney        | Wed 11-Jul-2018 08:00 (08:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Build%20WorkGroup+Meeting+2018-07-03&iso=20180703T22
* or http://www.wolframalpha.com/input/?i=10PM+UTC%2C+Jul+03%2C+2018+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1ZzWb6A1IcpC4NKc2NzhbLWiaxffjLWMrK36X7Wb2PCY/edit>

## Agenda

Extracted from **build-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/build

* policy: no use of PR code in live jobs [#1378](https://github.com/nodejs/build/issues/1378)
* Demo AWX (Ansible Tower) installation [#1340](https://github.com/nodejs/build/issues/1340)
* Request for elevated permissions [#1337](https://github.com/nodejs/build/issues/1337)
* Build WG self-nomination: amiller-gh [#1305](https://github.com/nodejs/build/issues/1305)
* request to join the build group [#1303](https://github.com/nodejs/build/issues/1303)
* State of Ansible [#1277](https://github.com/nodejs/build/issues/1277)
* Use of Docker in Build CI [#1226](https://github.com/nodejs/build/issues/1226)
* Release infrastructure post mortem [#1212](https://github.com/nodejs/build/issues/1212)
* Dropping 32-bit builds [#885](https://github.com/nodejs/build/issues/885)

## Invited

* Build team: @nodejs/build

## Notes

The agenda comes from issues labelled with `build-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: https://zoom.us/j/715960833
* For those who just want to watch: We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at https://www.youtube.com/c/nodejs+foundation/live when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.
* youtube admin page: https://www.youtube.com/my_live_events?filter=scheduled

**Comment by mhdawson** [comment]
I'm on holiday the 3rd.

**Comment by Trott** [comment]
@mhdawson Is "on the 7th" a typo? Should be "on the 3rd" instead?

**Comment by LucaLanziani** [comment]
Any update on this? Is the meeting happening today?

**Comment by mmarchini** [comment]
@nodejs/build please use 👍 and 👎 reactions on the [initial post](https://github.com/nodejs/build/issues/1387#issue-336200535) to indicate likely-to-attend-this-one and not-likely-to-be-able-to-attend respectively, so we can decide if we should have the meeting today or postpone it.

**Comment by refack** [comment]
I'm available, but it seems like there's is a majority of others who are unavailable, so I'm +1 on moving the date (rather then skipping the meeting).
Maybe the 5th or 6th?

**Comment by refack** [comment]
I'd say we officially postpone.

**Comment by Trott** [comment]
On the two requests to join the WG, would people like me to follow up with the two individuals? Or something else?

**Comment by refack** [comment]
I think @maclover7 tried to reach out, maybe you could too.
If you manage to schedule something, I'd be happy to join as well.

**Comment by rvagg** [comment]
Sorry, I should have indicated that I intended to join! Looks like it's been cancelled for us by a lack of Michaels presence to start it. Next week!

**Comment by mhdawson** [comment]
@nodejs/build I'd suggest the same time 6 EST on July 10th.  Lets see who can make that time

**Comment by Trott** [comment]
I could make it if it's an hour later, but not at that time.

**Comment by mhdawson** [comment]
Moving to July 10th.  @rvagg are you going to be able to make it?

If not we might see if we can do it earlier in the day.

**Comment by mhdawson** [comment]
I guess we'll have it at the original time 6pm EST. @Trott I can't make it an hour later, and we have 5 people who can make 6pm so figure its best to stick with that now even though we've not heard from Rod.

**Comment by Trott** [comment]
@mhdawson I believe Rod mentioned in the #node-build IRC channel today that he's on vacation.

**Comment by mhdawson** [comment]
@Trott thanks for the update, I think its too late to adjust time now even though.

**Comment by mhdawson** [comment]
Will start meeting in a few minutes.

**Comment by maclover7** [comment]
Thought I had put a thumbs down earlier, but I'm in the same boat as @Trott for today, could make it an hour later, but can't attend at 6

**Comment by mhdawson** [comment]
Meeting is live now

**Comment by joaocgreis** [comment]
Meeting recording: https://www.youtube.com/watch?v=rzyqeVzHCCg

**Comment by mhdawson** [comment]
@joaocgreis good reminder that we need to get the PR for the minutes out.

**Comment by maclover7** [comment]
PR for minutes: https://github.com/nodejs/build/pull/1435

**Comment by gdams** [comment]
closing as minutes now landed in: https://github.com/nodejs/build/commit/d121160b7d38cecd98c794a2a9549b89708a4754
